docs/system-integration.md

```diff
@@ -0,0 +1,82 @@
+!!! warning
+
+    These notes are a work-in-progress.
+    This page provides an overview as guidance only,
+    and not a series of definite instructions.
+    We plan to further expand on this information.
+
+## Audience
+
+This page is aimed at:
+
+* software developers
+* clinical data providers
+* system integrators
+
+who want to run their own installation of OpenSAFELY
+as a proof-of-concept trial prior to integrating into the live system.
```

Comment:
I would prefer not to split the sentence before and after the list. Perhaps:

```suggestion
This page is aimed at people who want to run their own installation of OpenSAFELY
as a proof-of-concept trial prior to integrating into the live system. This group is likely to be: 

* software developers
* clinical data providers
* system integrators


```

Reply:
I think this suggestion is better, but I'll just bump the extra sentence to a new line to try and preserve (roughly) semantic line breaks.